src/formats/onenote.ts

```diff
@@ -56,6 +57,14 @@ export class OneNoteImporter extends FormatImporter {
 				.onChange((value) => (this.importPreviouslyImported = !value))
 			);
 
+		new Setting(this.modal.contentEl)
```

Comment:
Since we're truncating to 50 characters, I don't think we need a setting to disable importing the alt text. Let's remove this please.